src/signTx.c

```diff
@@ -395,31 +388,36 @@ static void signTx_handleInputAPDU(uint8_t p2, uint8_t* wireDataBuffer, size_t w
 
 // ============================== OUTPUTS ==============================
 
+static void ui_displayAmount(uint64_t amount, ui_callback_fn_t callback)
```

Comment:
I guess we can pass header here. This way you can probably reuse it also for the fee ...